app/i18n/de/conf.php

```diff
@@ -105,7 +105,13 @@
 	),
 	'profile' => array(
 		'_' => 'Profil-Verwaltung',
-		'api' => 'API-Verwaltung',
+		'api' => array(
+			'_' => 'API-Verwaltung',
+			'check_link' => 'Check API status via: <kbd><a href="../api/" target="_blank">%s</a></kbd>',	// TODO
```

Comment:
Don't you want to do the German translation, @math-GH ?
Can also wait

Reply:
will do a separate PR for the German translation. More strings need a translation